private/tools/java/com/github/bazelbuild/rules_jvm_external/maven/MavenPublisherTest.java

```diff
@@ -0,0 +1,74 @@
+package com.github.bazelbuild.rules_jvm_external.maven;
+
+import static org.junit.Assert.assertEquals;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.nio.charset.StandardCharsets;
+import org.apache.maven.artifact.repository.metadata.Metadata;
+import org.apache.maven.artifact.repository.metadata.Versioning;
+import org.apache.maven.artifact.repository.metadata.io.xpp3.MetadataXpp3Reader;
+import org.apache.maven.artifact.repository.metadata.io.xpp3.MetadataXpp3Writer;
+import org.junit.Test;
+
+public class MavenPublisherTest {
+
+  @Test
+  public void testDeserialization() throws Exception {
+    String xml =
+        "<metadata modelVersion=\"1.1.0\">\n"
+            + "    <groupId>com.mycompany.app</groupId>\n"
+            + "    <artifactId>my-app</artifactId>\n"
+            + "    <versioning>\n"
+            + "        <latest>1.0</latest>\n"
+            + "        <release>1.0</release>\n"
+            + "        <versions>\n"
+            + "            <version>1.0</version>\n"
+            + "        </versions>\n"
+            + "        <lastUpdated>20200731090423</lastUpdated>\n"
+            + "    </versioning>\n"
+            + "</metadata>";
+    Metadata metadata =
```

Comment:
I'm curious. This looks like it's checking that the maven library works, but the test names suggests that this is about `MavenPublisher`.

Reply:
We could just remove these tests, they were used early on in development just to make sure the library worked how we expected.